docs/en/ingest-management/elastic-agent/run-container-common/deploy-elastic-agent.asciidoc

```diff
@@ -17,5 +17,9 @@ elastic-agent-fj2z9             1/1     Running   0          81m
 elastic-agent-hs4pb             1/1     Running   0          81m
 ------------------------------------------------
 
-
-
+[TIP] 
+.Running {agent} on a read-only file system
+==== 
+If you'd like to run {agent} on Kubernetes on a read-only file 
+system, you can do so by specifying the `readOnlyRootFilesystem` option.
```

Comment:
I think this is good enough, I don't know if we should really go into details on how to configure the pod specification. Any kubernetes engineer understands this option and its easily search-able to find examples on how this is done.

Optional adding it as a commented out section in our kubernetes manifests, would be a better way of doing this.

Reply:
Sounds good. I've left it as is.